### PR TITLE
[Core] Adding OSUtilities

### PR DESCRIPTION
**Pull request by loumalouomega**
This is part of #4906. I separate this in order to make easier the review

### CODE REVIEW

File: kratos/utilities/os_utilities.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+#include <stdio.h>  /* defines FILENAME_MAX */
+#ifdef _WIN32
```
Comment (philbucher) [inline kratos/utilities/os_utilities.cpp]: There is a macro for this at the top of define.h

Comment (loumalouomega) [reply kratos/utilities/os_utilities.cpp]: Done, I didn't know it

**Comment by loumalouomega** [comment]
Closing as merged on #4911

**Comment by philbucher** [comment]
hm I wanted to confirm with @roigcarlo if this is portable but now its too late and failing :/

Please be careful with os-specific stuff in the core

**Comment by loumalouomega** [comment]
I merged #4911 without remembering that this was included

**Comment by loumalouomega** [comment]
Sorry

**Pull request closed without merging**